vNext/extensions/applicationinsights-properties-js/src/PropertiesPlugin.ts

```diff
@@ -47,6 +48,8 @@ export default class PropertiesPlugin implements ITelemetryPlugin, IPropertiesPl
 
         this._logger = core.logger;
         this.context = new TelemetryContext(core.logger, this._extensionConfig);
+        this._breezeChannel = Util.getExtension(extensions, BreezeChannelIdentifier);
+        this.context.appId = () => this._breezeChannel ? this._breezeChannel["_appId"] : null;
```

Comment:
Is Breeze channel appId available during initialization?

Reply:
Only if it is provided in config. Otherwise, it is only available after sending telemetry the first time.